.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 100,
```

Comment:
change to 80 please

Reply:
sure, if you want me to switch the staged files over to the add script you pointed out, i can change that too while im at it